### DISCUSSION TITLE
Locales for Mozilla Firefox

### DISCUSSION THREAD

**Discussion by alkisg** [microsoft/winget-pkgs#35387] [Q&A]
Hi, while trying to install Firefox in my Greek locale, I got the English version instead.
In https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Mozilla/Firefox/94.0 I can see that some locales other than English are available, but not all the ones that Mozilla offers at https://download-installer.cdn.mozilla.net/pub/firefox/releases/94.0/win64/.

Wouldn't it be better to support all the available locales?
Do they need to be implemented separately?
Should we file bug reports for each one of them?

**Comment by KaranKad** [comment]
It would definetly be better to support all locales but there are too many of them, each locale as five different types of installer.
exe - x64, msi - x64, exe - arch64. exe - x86, msi - x86. So, currently we only add different locales as requested by users, I'll add greek one.

Also, can you provide translation for this in greek?
`Mozilla Firefox is free and open source software, built by a community of thousands from all over the world.`

**Comment by alkisg** [comment]
Thank you @KaranKad!

The official translation that also appears in Firefox > About, is:

> Το Mozilla Firefox είναι ελεύθερο λογισμικό ανοικτού κώδικα που δημιουργείται από μια κοινότητα χιλιάδων ατόμων από όλο τον κόσμο.

**Comment by mtengmo** [comment]
Is it possible to add the swedish one?
